api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -562,6 +563,62 @@ public PasswordAuthentication getPasswordAuthentication() {
 	 *
 	 * @return the ServiceContext
 	 */
+
+	public static   ServiceContext getserviceObject(){
+		ApplicationContext s_context = new ClassPathXmlApplicationContext("spring.xml");
```

Comment:
I would think this class needs to be application context aware, instead of creating a new application context object because there might be one already.